src/Models/Elements/Element.cs

```diff
@@ -18,6 +18,18 @@ public Element()
             validationResult = new ValidationResult();
         }
 
+        public Element(IElement element)
+        {
+            Type = element.Type;
+            Properties = element.Properties;
+            validationResult = new ValidationResult();
+        }
+
+        public Element Clone()
+        {
+            return new(this);
+        }
```

Comment:
Can this become a fat arrow method?

Reply:
I've since removed this as it was an experiment that didn't work and I'd missed it